src/subscription_manager/i18n.py

```diff
@@ -123,7 +123,7 @@ def _find_lang_alternative(cls, language):
             try:
                 lang = gettext.translation(APP, DIR, languages=[new_language])
             except IOError as err:
-                log.info('Could not import locale either for %s: %s' % (new_language, err))
+                log.error('Could not import locale either for %s: %s' % (new_language, err))
```

Comment:
I don't think it is necessary to have `error` here. `debug` would be sufficient. subscription-manager is not translated to most of languages (locales) of this world. BTW: when some unsupported locate is used on the system, then following message was printed to the console for most of commands: `No handlers could be found for logger "subscription_manager.i18n"`